tsk/pool/lvm_pool_compat.cpp

```diff
@@ -145,8 +145,9 @@ lvm_logical_volume_img_read(TSK_IMG_INFO * img_info, TSK_OFF_T offset, char *buf
     IMG_POOL_INFO *pool_img_info = (IMG_POOL_INFO *)img_info;
     libvslvm_error_t *vslvm_error = NULL;
 
-    // correct the offset to be relative to the start of the logical volume
-    offset -= pool_img_info->pool_info->img_offset;
+    // Commented: offset is already relative (0) to the img_info, that is relative to the volume
+    //// correct the offset to be relative to the start of the logical volume
+    //offset -= pool_img_info->pool_info->img_offset;
```

Comment:
This commit is crucial for the LVM to work on scenarios with pool offset. I have tested on images with and without pool offset with success.

libvslvm is already generating the img_info relative to the volume, and we are passing this img_info and offset=0 to tsk_fs_open_img and others. 

Doing this step of turning relative what is already relative is the cause of the errors we've seen.  

I just commented the lines, but with your approval, I would like to remove them, to clean up the code.